egs2/TEMPLATE/asr1/db.sh

```diff
@@ -17,6 +17,7 @@ DIRHA_WSJ_PROCESSED="${PWD}/data/local/dirha_wsj_processed"  # Output file path
 DNS=
 DNS2=
 DNS3=
+DNS4="${PWD}/data_dns4_raw"
```

Comment:
It's better to use `DNS4` or `downloads` here to follow the convention instead of data_dns4_raw

Reply:
Ok! will change to `downloads`